.codesandbox/sandbox/package.json

```
@@ -5,6 +5,7 @@
   "main": "index.html",
   "scripts": {
     "start": "parcel index.html --open",
+    "start-e2e": "parcel end-to-end-test.html --open",
```

I'm a bit reluctant to add this in the first place, since it's not something you'll usually want to run. Either way, this command does not start the actual end-to-end tests, so let's at least not imply that through its name :)

```suggestion
    "start-e2e-host": "parcel end-to-end-test.html --open",
```

---

I think it's no harm adding this here, even if it's not usually run, as it's good to point out that 'something different' is available to be run. Tweaked for webpack now, and tweaked the HTML a little too - 7abf9f84